### PR TITLE
Feature/get content watch interval from config file

### PR DESCRIPTION
**Pull request by santiagonoguez**

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -5,6 +5,20 @@ const DEFAULT_OFFSET = 1;
...
+reset();
+
+function init() {
+  return common.getDisplayProperty('contentwatchinterval')
+  .then(value => {
+    if (typeof value !== 'string') {
+      return;
+    }
+
+    contentWatchInterval = Math.max(0, Number(value)) * MINUTES;
```
Comment (tejohnso) [inline src/config.js]: Don't we need to make sure there's a reasonable minimum?

Comment (santiagonoguez) [reply src/config.js]: @tejohnso good idea. I'll limit it to no less than half regular watchdog interval. 

0 is still a valid value for disabled state.

Comment (tejohnso) [reply src/config.js]: Zero might be okay for disabled state but only if it's explicitly specified in the display properties file. 
A display properties file with no `contentwatchinterval` line shouldn't default to 0

Comment (santiagonoguez) [reply src/config.js]: @tejohnso That's exactly how it works now, if contentwatchinterval parameter is not found then it stops executing here:

https://github.com/Rise-Vision/watchdog-module/blob/feature/get_content_watch_interval_from_config_file/src/config.js#L16

and so it defaults to an interval that's the same as the regular watch interval:

https://github.com/Rise-Vision/watchdog-module/blob/feature/get_content_watch_interval_from_config_file/src/config.js#L10

**Pull request merged**